src/cuttlefish_conf.erl

```diff
@@ -67,13 +67,11 @@ files(ListOfConfFiles) ->
 
 -spec file(file:name()) -> conf() | cuttlefish_error:errorlist().
 file(Filename) ->
-    case conf_parse:file(Filename) of
+    case hocon:load(Filename, #{format => proplists}) of
```

Comment:
cannot use `convert => [duration]` here, because cuttlefish have default values like
```
{mapping, "management.listener.http.send_timeout", "emqx_management.listeners", [
  {datatype, {duration, ms}},
  {default, "15s"}
]}.
```
and we have to convert these after `hocon:load`.